### ISSUE TITLE
VisIt restarts a died engine with a serial engine when run in nowin mode.

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1036]
When you run visit under the cli to produce a movie in "nowin" mode, it used to be that if the engine died (either with a crash or running out of time in a batch system), visit would restart the engine using the same parameters as it was running before it crashed (e.g. if it was running in parallel with N cpus in the batch system, it would start a new engine in the batch system with N cpus). In 2.0 it starts a serial engine.

This is a very serious problem for those making movies in a batch system.

Here is command that reproduces it, along with the script used.

visit -v 2.0 -cli -nowin -s visit_script.py

visit_script.py contains:
OpenComputeEngine("localhost", ("-l", "msub/srun", "-t", "00:00:30", "-np",
"16", "-nn", "8", "-q", "pbatch", "-A", "bdivp"))
OpenDatabase("localhost:/usr/gapps/visit/data/wave*.silo database", 0)
AddPlot("Pseudocolor", "pressure", 1, 1)
DrawPlots()
while 1:
   SaveWindow()
   TimeSliderNextState()

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Eric Brugger
Original creation: 06/30/2010 12:30 pm
Original update: 07/13/2010 04:31 pm
Ticket number: 191
